modules/nf-core/samtools/index/tests/csi.nextflow.config

```diff
@@ -0,0 +1,7 @@
+process {
+
+    withName: SAMTOOLS_INDEX {
+        ext.args = '-c'
```

Comment:
why this change here?  or is it part of modules update?

Reply:
it is part of the modules update, it is a test